config/gradle/common.gradle

```diff
@@ -84,8 +93,9 @@ test {
     }
 
     jacoco {
-        append = false
-        excludes = ["org.terasology.protobuf.*"]
+       // append = false
```

Comment:
is this still required? if not, please remove, else it should not be commented out